tests/test_auth.py

```diff
@@ -0,0 +1,14 @@
+import unittest
+from unittest import mock
+from src.cisco_gnmi.auth import CiscoAuthPlugin
+
+
+def test_call():
```

Comment:
Need to evaluate this further, might be fine for now.
Ideally we mock a gRPC server and ensure that login metadata is being passed with each call.